File: app/src/main/java/com/example/proyecto/domain/Util/New/Pruebas/RetrofitHelper.java
package com.example.proyecto.domain.Util.New.Pruebas;

import com.example.proyecto.domain.Util.Util;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitHelper {
    public static Retrofit retrofit;
    public static Retrofit getRetrofit(){
        retrofit = new Retrofit.Builder()
                .baseUrl(Util.RUTA)
                .addConverterFactory(GsonConverterFactory.create())
                .build();
        return retrofit;
    }
}

File: app/src/main/java/com/example/proyecto/core/RetrofitHelper.kt
package com.example.proyecto.core

import com.example.proyecto.domain.Util.Util
import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory
import android.util.Log

object RetrofitHelper {
    fun getRetrofit(): Retrofit {
        return Retrofit.Builder()
            .baseUrl(Util.RUTA)
            .addConverterFactory(GsonConverterFactory.create())
            .build()
    }
    fun getRetrofit_API(): Retrofit {
        val baseUrl = Util.RUTA_API
        Log.d("RetrofitHelper", "Base URL: $baseUrl") // Agrega esta línea para imprimir la URL en el log

        return Retrofit.Builder()
                .baseUrl(baseUrl)
                .addConverterFactory(GsonConverterFactory.create())
                .build()
    }

}